app/styles/_vars.scss

```diff
@@ -1,10 +1,3 @@
-// VARS
-$base-font: 16;
-$line-height: 24;
-$transition-timing: 250ms;
-$transition-timing-queue: 150ms;
-$spacing: 1.5rem;
-
 // MIXINS and EXTENDS
```

Comment:
Oh oh oh ^ 

These were already in the wqxr client project… I thought this was something you added. Sorry for the misunderstanding, so let's just leave these where they are in `_vars` (so don't delete these)

Reply:
put back in